app/templates/components/explore/side-bar.hbs

```diff
@@ -0,0 +1,69 @@
+<div class="ui fluid vertical menu">
+  <div class="map item">
+    {{g-map lat=37.7833 lng=-122.4167 zoom=2 class="explore"}}
+  </div>
+  <div class="item">
+    <div class="ui input">
+      <input type="text" placeholder="enter location">
+    </div>
+  </div>
+  <div class="item">
+    {{#ui-accordion}}
+      <span class="title">{{t 'Categories'}}</span>
```

Comment:
Add `<i class="dropdown icon"></i>` to all the title spans. Put it before the text so that the icon is displayed at the start